src/Nox.Types/Nox.Types.csproj

```diff
@@ -24,6 +24,7 @@
   <ItemGroup>
     <None Include="..\..\images\nox.png" Pack="true" PackagePath="" />
     <InternalsVisibleTo Include="Nox.Types.Tests" />
+	  <InternalsVisibleTo Include="Nox.EntityFramework.SqlServer" />
```

Comment:
this should not be needed?

Reply:
Property Weight.QuantityValueDecimalPrecision, and classes WeightConversion, WeightUnit were marked as internal, hence that was needed. But I've marked them now as public and removed this line as we discussed.